app/views/ErrorPage.scala.html

```diff
@@ -15,90 +15,85 @@
 
-    @isCurrentTaxYear match {
-        case _ if(code >= 0) => {<h1 id="title" class="heading-xlarge">@messages("ServiceMessage." + (code) + ".h1")</h1>}
-        case utils.FormMappingsConstants.CY => {<h1 id="title" class="heading-xlarge">@messages(pageHeading,taxYearRange.cyminus1.toString, taxYearRange.cy.toString)</h1>}
-        case utils.FormMappingsConstants.CYP1 => {<h1 id="title" class="heading-xlarge">@messages(pageHeading, taxYearRange.cy.toString, taxYearRange.cyplus1.toString)</h1>}
-        case _ => {<h1 id="title" class="heading-xlarge">@messages(errorMessage + "", "", "")</h1>}
-    }
+    <h1 id="title" class="govuk-heading-xl">
+        @isCurrentTaxYear match{
+            case _ if(code >= 0) => { @messages("ServiceMessage." + (code) + ".h1") }
+            case utils.FormMappingsConstants.CY => { @messages(pageHeading,taxYearRange.cyminus1.toString, taxYearRange.cy.toString) }
+            case utils.FormMappingsConstants.CYP1 => { @messages(pageHeading, taxYearRange.cy.toString, taxYearRange.cyplus1.toString) }
+            case _ => { @messages(errorMessage) }
+        }
+    </h1>
 
-    <p class="lede">
+    <p class="govuk-body-l">
      @isCurrentTaxYear match {
             case _ if(code == 63091 || code == 65127 || code == 63085) => {
-
                 @Html(messages("ServiceMessage." + (code)))
-
             }
             case _ if(code == 63088 || code == 64989 || code == 10003)  => {
-
                 @Html(messages("ServiceMessage." + (code) + ".1"))
-
             }
-
-            case utils.FormMappingsConstants.CY => {@Html(messages(errorMessage + isCurrentTaxYear , taxYearRange.cyminus1.toString, taxYearRange.cy.toString))}
-
-            case utils.FormMappingsConstants.CYP1 => {@Html(messages(errorMessage + isCurrentTaxYear , taxYearRange.cy.toString, taxYearRange.cyplus1.toString))}
-
+            case utils.FormMappingsConstants.CY => {@Html(messages(errorMessage, taxYearRange.cyminus1.toString, taxYearRange.cy.toString))}
+            case utils.FormMappingsConstants.CYP1 => {@Html(messages(errorMessage, taxYearRange.cy.toString, taxYearRange.cyplus1.toString))}
             case _ => {@Html(messages(errorMessage))}
-
         }
     </p>
 
+    @if(code == 63088 || code == 64989 || code == 10003) {
+        <p class="govuk-body">
+            @Html(messages("ServiceMessage." + (code) + ".2"))
+        </p>
+    }
 
-        @if(code == 63088 || code == 64989 || code == 10003) {
-            <p>
-                @Html(messages("ServiceMessage." + (code) + ".2"))
-            </p>
+    <div class="govuk-form-group">
+        @code match {
+            case 63082 => {<a class="govuk-link" id="link-try-again" href='javascript: window.history.go(-1)'>@messages("Service.tryagain")</a>}
+            case 63090 => {<a class="govuk-link" id="link-add-another" href='javascript: window.history.go(-1)'>@messages("Service.excludeanotheremployee")</a>}
```

Comment:
no differences on lines 73 and 74..so plz make this into one line using |

Reply:
good point